src/main/java/games/strategy/triplea/util/WrappedInvocationHandler.java

```diff
@@ -57,6 +58,7 @@ public Object invoke(final Object proxy, final Method method, final Object[] arg
     if (shouldHandle(method, args)) {
       return handle(method, args);
     }
-    throw new IllegalStateException("not configured");
+    throw new IllegalStateException(String.format("not configured for method %s, args: %s",
+        method.toString(), Arrays.asList(args).toString()));
```

Comment:
Nit: You could just call `Arrays.toString(args)` here.  Also, I believe the explicit calls `Object#toString()` are unnecessary as `String#format()` does that implicitly.

Reply:
Thanks, suggestion accepted, simplified in: da81c5d